stores/json/store.go

```diff
@@ -186,29 +186,35 @@ func (store Store) encodeValue(v interface{}) ([]byte, error) {
 
 func (store Store) encodeArray(array []interface{}) ([]byte, error) {
 	out := "["
-	for i, item := range array {
+	empty := true
+	for _, item := range array {
 		if _, ok := item.(sops.Comment); ok {
 			continue
 		}
+		if !empty {
```

Comment:
We no longer need to skip the last item in the array? `if i != len(array)-1 {`

Reply:
Yes, because the comma is inserted before an item is added. The `empty` flag keeps track whether an item has been added already (if `empty==true` none has been added, so no comma is needed).